public/modules/rewrite/my_bodies/edit_circles.html

```diff
@@ -0,0 +1,21 @@
+<table class="edit-circles-table">
```

Comment:
Liked this view for the small amount currently present, needs some multi user management later on though.